docs/css/style.css

```diff
@@ -180,15 +180,18 @@ a.direct-link {
 
 ul {
   box-sizing: content-box;
-  column-count: 2;
-  column-gap: 30px;
   margin-top: 20px;
   padding: 0 15px;
 }
 
+h2#table-of-contents + ul,
```

Comment:
[Refactor] The `table-of-contents` ID is auto-generated. We won't want to take a dependency on it existing as-is. It's also kind of coincidental to assume the list immediately following the h2 table of contents should be two-column.

```suggestion
```

Could you add the `two-column` class to the list instead? I.e. add another `{:.two-column}`?

Reply:
Well noticed, thanks! Fixed on [64d8788](https://github.com/mochajs/mocha/pull/5110/commits/64d8788f01a4348a9c7788a39fe2b79e2bef5936)